README.md

```diff
@@ -17,7 +17,9 @@ To add a conference to the list, please [contribute](#contributing)!
 | [Data Governance Summit 2018](http://datagovernance.in/index.html) | 8 Feb | JW Marriott, Mumbai | The conference which give insights of the current challenges being faced regarding data quality and security. |
 | [Hosting and Domain Conference India](http://www.hdcon.org/) | 3 Feb | Hyderabad | Hosting and Domain Conference is the World's Top initiative to discuss on hosting, domain, blogging and web technologies. |
 | [ReactFoo Pune](https://reactfoo.in/) | 19-20 Jan | Pune | A conference on the lessons learned while implementing React and React Native. |
-| [IN/Clojure](http://inclojure.org/) | 12-13 Jan | Bengaluru | IN/Clojure is India's annual Clojure conference. It is also the only Clojure conference in all of Asia.
+| [IN/Clojure](http://inclojure.org/) | 12-13 Jan | Bengaluru | IN/Clojure is India's annual Clojure conference. It is also the only Clojure conference in all of Asia. |
+| [Times AI Hub Artificial Intelligence Conclave](https://www.timesaihub.com/) | 16 Mar | Bengaluru | It is India’s largest Artificial Intelligence conclave wherein professionals from the AI startups, automation companies, freelancers, AI enthusiasts can come on a common platform to discuss "THE PRACTICAL IMPLICATIONS OF AI FOR ENTERPRISE ORGANIZATIONS" & "THE SOLUTIONS THAT ARE TRANSFORMING BUSINESS PRODUCTIVITY"
```

Comment:
The list is in the reverse chronological order. Please add this entry between GopherCon India and Great Indian Developer Summit.

Reply:
For the description - "Practical Implications of AI for enterprise organizations and Solutions transforming business productivity are discussed".

Reply:
For the location - IISC Bengaluru. Instead of just Bengaluru.